app/models/parcel.rb

```diff
@@ -281,6 +282,10 @@ def opened_financial_year?
     FinancialYear.on(given_at)&.opened?
   end
 
+  def financial_year_in_closure_preparation?
```

Comment:
Can this be named more explicitly? Something that makes it clear how we're finding the financial year. Maybe `given_during_closure_preparation?` ?